src/redisearch_rs/inverted_index/src/index_result.rs

```diff
@@ -719,12 +719,28 @@ impl<'index> RSIndexResult<'index> {
         }
     }
 
+    /// Resets the metadata attached to this [`RSIndexResult`].
+    ///
+    /// This [`RSIndexResult`] is assumed to be [`Virtual`] with no
+    /// [`RSDocumentMetadata`] embedded in it.
+    ///
+    /// [`Virtual`]: RSResultData::Virtual
```

Comment:
What is this line for?

Reply:
Rust doc comments are markdown.
This is defining the link for [`Virtual`] given that symbol is not in scope.